### PR TITLE
Bump embedded-hal to rc 2

### PR DESCRIPTION
**Pull request by CBJamo**

### CODE REVIEW

File: src/spi.rs [hunk 1]
```diff
@@ -220,9 +220,9 @@ mod embedded_hal_impl {
                         };
                         transfers.push(SpidevTransfer::read_write(tx, buf));
                     }
-                    SpiOperation::DelayUs(us) => {
-                        let us = (*us).try_into().unwrap_or(u16::MAX);
-                        transfers.push(SpidevTransfer::delay(us));
+                    SpiOperation::DelayNs(ns) => {
+                        let ns = (*ns).try_into().unwrap_or(u16::MAX);
+                        transfers.push(SpidevTransfer::delay(ns));
```
Comment (eldruin) [inline src/spi.rs:225]: The nanoseconds need to be converted to microseconds here rounding up to at least 1 if not 0

**Review by eldruin — CHANGES_REQUESTED** [review]
Thanks! The conversion is missing, though.
Could you also add an entry to the changelog about the update to embedded-hal?

File: src/spi.rs (hunk 1 above)
Comment (CBJamo) [reply src/spi.rs]: Whoops, of course. Fixed up.

**Review by eldruin — APPROVED** [review]
Great, thank you!

**Pull request merged**